.github/workflows/wasm-ci.yml

```diff
@@ -20,19 +20,19 @@ jobs:
     - name: Install emsdk
       run: tools/install-wasm-dependencies
 
-    - name: Cache internal dependencies
-      id: internal_cache
-      uses: actions/cache@v1.1.2
-      with:
-        path: build/local
-        key: ${{ runner.os }}-internal-${{ hashFiles('tools/install-dependencies') }}
+    # - name: Cache internal dependencies
```

Comment:
Let's not just comment out chunks of code. We can either remove this block completely, or disable it through some option

Reply:
will revert, this was for testing CI